### PR TITLE
Added new fields in Inventory table and Flyout Details

### LINKED ISSUE #3317: Include the detection timestamp, severity, and score of the CVEs in the vulnerabilities inventory as well as the scans timestamps
**Describe the solution you'd like**
As part of the epic https://github.com/wazuh/wazuh/issues/8727, we are adding more information to the inventory of vulnerabilities that affect each agent. This information should be displayed in the Wazuh UI, in the CVEs inventory. The information consists of the next set of items:

- **Severity**: It should be a new column in the inventory and the value is a string is limited to one of the next possible values. `Critical`, `High`, `Medium`, `Low`, `None`, and `-`. It would be desirable for the users to have the ability of filtering by severity.
- **CVSS2** and **CVSS3**: These should be two new columns in the inventory that represent a numerical score of the severity. This is a score ranging from 0.0 to 10.0. It would be desirable for the users to have the ability of filtering by severity score ranges.
- **Detection time**: This is a timestamp that represents the first time that the vulnerability was identified in a Vulnerability Detector scan.

Apart from that, it is required to have information in the header of the inventory that provides details related to:

- The timestamp of the last Full scan of the Vulnerability Detector performed over the agent.
- The timestamp of the last Partial scan of the Vulnerability Detector performed over the agent.

For all the information, you can make use of the API endpoints developed as part of the issue https://github.com/wazuh/wazuh/issues/8863.

### PR DESCRIPTION
**Pull request by Machi3mfl**
Hi Team,
This PR includes the detection timestamp, severity, and score of the CVEs in the vulnerabilities inventory and the scans timestamps.

Close #3317

### CODE REVIEW

**Comment by Machi3mfl** [comment]
Screen Shot 2021-08-02 at 17 19 01
Screen Shot 2021-08-02 at 17 19 17

File: public/components/agents/vuls/inventory/flyout.tsx [hunk 1]
```diff
@@ -88,10 +98,14 @@ export class FlyoutDetail extends Component {
     }
   }
 
+
+
```
Comment (frankeros) [inline public/components/agents/vuls/inventory/flyout.tsx:102]: remove unnecessary line breaks

File: public/components/agents/vuls/inventory/flyout.tsx [hunk 2]
```diff
@@ -55,18 +56,27 @@ export class FlyoutDetail extends Component {
...
-      const currentItem = this.props.item;
+      let currentItem = this.props.item;
 
       if (!currentItem) {
         throw false;
       }
+
+      let lastScan = await this.getLastScan();
+      currentItem = { ...currentItem, ...lastScan};
+
```
Comment (frankeros) [inline public/components/agents/vuls/inventory/flyout.tsx:79]: if you need a clone of the object you could use a const too
```js
     const currentItem = {...this.props.item};

      if (!currentItem) {
        throw false;
      }

      const lastScan = await this.getLastScan();
      Object.assign(currentItem, {...lastScan});
```

File: public/components/agents/vuls/inventory/table.tsx [hunk 3]
```diff
@@ -125,13 +130,38 @@ export class InventoryTable extends Component {
...
+      },
+      {
+        field: 'severity',
+        name: 'Severity',
+        sortable: true,
+        width: `${width}`
+      },
+      {
+        field: 'cvss2_score',
+        name: 'CVSS2',
```
Comment (frankeros) [inline public/components/agents/vuls/inventory/table.tsx:142]: `name: 'CVSS2 Score'`

File: public/components/agents/vuls/inventory/table.tsx [hunk 4]
```diff
@@ -125,13 +130,38 @@ export class InventoryTable extends Component {
...
+        sortable: true,
+        width: `${width}`
+      },
+      {
+        field: 'cvss3_score',
+        name: 'CVSS3',
```
Comment (frankeros) [inline public/components/agents/vuls/inventory/table.tsx:148]: `name: 'CVSS3 Score'`

**Review by frankeros — APPROVED** [review]

**Comment by Machi3mfl** [comment]
This PR depends on this API [Epic](https://github.com/wazuh/wazuh/issues/8727) issue. 
The API solution is implemented in this [branch](https://github.com/wazuh/wazuh/tree/dev-8727-cves-inventory-improvements).

**Pull request merged**